### DISCUSSION TITLE
SQL views exists in nocodb ? [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2145] [Discord Threads]
> 🙋‍♂️ **martin_michelle**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You can create it using other tools and then you can run meta sync on nocodb  
#  
> 💁 **o1lab**  
#### what is your usecase  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/sync-schema  
#  
> 🙋‍♂️ **martin_michelle**  
#### i'm migrating from AirTable and I have 100+ Formulas, and i'm getting a lot of errors while re-creating the formulas in nocodb, so i going to create formulas as a SELECT Statment inside SQL view  
#  
> 💁 **o1lab**  
#### ahok  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975840640623603762)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
